sdk/tests/core/test_az_logging.c

```diff
@@ -84,6 +84,29 @@ static void _log_listener_NULL(az_log_classification classification, az_span mes
   }
 }
 
+static void _log_listener_no_op(az_log_classification classification, az_span message)
```

Comment:
doesn't look like a no-op to me. Please change name?

Reply:
Or remove most of the body, so it is truly a no-op. And if it is used only in test_az_log_corrupted_response(), move it below test_az_log(), but above test_az_log_corrupted_response().

Reply:
"`log_listener_non_corrupted_response()`"?